test/passing/break_cases_all.ml

```diff
@@ -132,3 +132,15 @@ let () =
   | foooooo
```

Comment:
The symlinks and .ref/.opts trick is worth setting up here I think.

Reply:
done